apps/src/codebridge/CodebridgeRegistry.ts

```diff
@@ -28,4 +32,16 @@ export default class CodebridgeRegistry {
   public getConsoleManager() {
     return this.consoleManager;
   }
+
+  public setNeighborhood(neighborhood: Neighborhood) {
+    this.neighborhood = neighborhood;
+  }
+
+  public getNeighborhood() {
+    if (this.neighborhood) {
```

Comment:
I think we can just `return this.neighborhood`, if it's null it will return null